data/redirects.toml

```diff
@@ -28,6 +28,11 @@ from = "/almanack/"
 to = "https://almanack.data.spotlightpa.org"
 code = 302
 
+[[redirects]]
+from = "/about/statecollege/"
+to = "/support/state-college"
+code = 302
```

Comment:
You can also put add an aliases key to the support/state-college page, which is a little easier to keep track of. These redirects are mostly for things that redirect off the site. (/events/ used to redirect off the site, but doesn't anymore.)